code/src/components/Albums.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import Artist from './Artist';
+import Icons from './Icons';
+import '../album.css';
+
+// const Albums function = Defining the functional component Albums that takes in a props object.
```

Comment:
Well commented in your code - this helps us a lot when doing the code review :)
/Ylva & Emma